src/core/lib/security/authorization/evaluate_args.cc

```diff
@@ -83,8 +138,7 @@ absl::string_view EvaluateArgs::GetSpiffeId() const {
     return spiffe_id;
   }
   if (grpc_auth_property_iterator_next(&it) != nullptr) return spiffe_id;
-  spiffe_id = absl::string_view(
-      reinterpret_cast<const char*>(prop->value, prop->value_length));
+  spiffe_id = absl::string_view(reinterpret_cast<const char*>(prop->value));
```

Comment:
I missed this in the earlier PR. Could you make these additional changes. 
I don't think we need the reinterpret_cast here.
This could be absl::string_view spiffe_id = absl::string_view(prop->value, prop->value_length) or better to directly return.
And maybe reformat this implementation such that the line 135-140 are in if(cond || cond || cond) return "". Since we anyway aren't logging.

Reply:
Thank you for the review. I have made the requested change.